submissions/israelOladosu/php/hobbies.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+$name = ['Adaa',  'Afolakemi',  'Tox',  'Emmanuel',  'Nifemi',  'Mary',  'Israel',  'Opeyemi'];
+
+$hobbies = ['coding',  'signing',  'skating',  'football',  'tennis',  'dancing',  'drumming',  'reading'];
+
+function display($array1, $array2)
+{
+    for ($i = 0; $i <= count($array1) - 1; $i++) {
+        $firstName = $array1[$i];
+        $hobby = $array2[$i];
+        echo "My name is $firstName. I love $hobby." . '<br >';
```

Comment:
Same thing. Instead of using `<br>` kindly use `\n`  - which mean `new line`
echo "My name is $firstName. I love $hobby. \n";

Reply:
Really, it was displaying on  a new in my browser.

I will do that now. Thanks

Reply:
Done now.